src/base.jl

```diff
@@ -49,6 +49,16 @@ end
 @inline MPS(A::AbstractArray, ::Val{:right}, Dcut::Int, args...) = _left_sweep_SVD(A, Dcut, args...)
 @inline MPS(A::AbstractArray, ::Val{:left}, Dcut::Int, args...) = _right_sweep_SVD(A, Dcut, args...)
 
+@inline Base.dropdims(ψ::MPS, i::Int) = (dropdims(A, dims=i) for A ∈ ψ)
+@inline Base.dropdims(ψ::MPS) = Base.dropdims(ψ, 2)
+
+function idMPS(::Type{T}, L::Int) where {T <: Number}
```

Comment:
Thus far, in contract and boundary MPS, but will be in more places. Ideally, it would be nice to have something like `MPS(I)` or maybe we should initialize `MPS` with ones. Then calling `MPS(T, L)` would be enough.  I think that both are valid solutions.

Reply:
I'll add something like this after the merge